include/circt/Dialect/SV/SVAttributes.td

```diff
@@ -0,0 +1,33 @@
+//===- SVAttributes.td - Attributes for HW dialect ---------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines SystemVerilog dialect-specific attributes.
+//
+//===----------------------------------------------------------------------===//
+
+def AttributeAttr : AttrDef<SVDialect, "Attribute"> {
```

Comment:
This name won't fly, it will just be too confusing.  Maybe SVAttributeAttr?

Reply:
Agree. I cringed everytime I wrote this. 😂  `SVAttributeAttr` is the right choice and follows things like `FModuleOp` where there are name collisions that need to be uniqued by the dialect. 👍

Reply:
Changed in a `!fixup` commit.